qt/DetailsDialog.cc

```diff
@@ -1289,38 +1290,64 @@ void DetailsDialog::onTrackerSelectionChanged()
 void DetailsDialog::onAddTrackerClicked()
 {
     bool ok = false;
-    QString const url = QInputDialog::getText(this, tr("Add URL "), tr("Add tracker announce URL:"), QLineEdit::Normal,
-        QString(), &ok);
+    QString const urls = QInputDialog::getMultiLineText(this, tr("Add URL(s) "),
+        tr("Add tracker announce URL(s) (one per line):"), {}, &ok);
```

Comment:
I don't think that end needs to be parenthetical. How about:

```suggestion
        tr("Add tracker announce URLs, one per line:"), {}, &ok);
```

Reply:
@ckerr I will commit these inline suggestions directly here, if you need them to be squashed, and this PR rebased with them, let me know

Reply:
Squashing and rebasing after review would be good.

Reply:
GitHub has this nice, handy "squash and merge" button that I can use when merging it into master, so no need to worry about the commit history in your branch IMO